website/docs/reference/datasources/querying-mysql.md

```diff
@@ -53,15 +53,39 @@ MySQL databases can be queried using the standard [SQL syntax](https://dev.mysql
 
 ![](/img/postgres.gif)
 
-## Using Prepared Statement (Beta)
 
-Normal query execution simply string concatenates the evaluated values of the javascript bindings to produce the final query. This opens up the possibility of SQL injection by merging untrusted user input with trusted data for execution. Using a prepared Statement is one strategy for mitigating this risk.
 
-Appsmith converts the user query into a parameterized one by replacing the bindings in the query with '?'. The payload is then inserted one by one ensuring that the bindings get properly escaped and sanitized before the query is sent to the database for execution.
+### SQL modes
+
+The SQL mode (```sql_mode```) is a system variable in MySQL that controls the behavior of the MySQL server. The SQL mode can be used to configure the server to be strict or forgiving when accepting input data, enable or disable standard SQL conformance, or provide better compatibility with other databases. [Strict mode](https://dev.mysql.com/doc/refman/8.0/en/sql-mode.html#sql-mode-strict) can help you ensure the integrity of your data by preventing the database from automatically inserting default values for missing or invalid data. 
+
+#### How to use SQL modes
+
+To change the SQL mode at runtime, you can use a **SET** statement to set the global or session ```sql_mode``` system variable:
+
+```js
+SET GLOBAL sql_mode = 'mode1,mode2,...';
+SET SESSION sql_mode = 'mode1,mode2,...';
+```
+where **mode1**, **mode2**, etc. are the specific [SQL modes](https://dev.mysql.com/doc/refman/8.0/en/sql-mode.html#sql-mode-important) that you want to enable or disable.
```

Comment:
I noticed we are hyperlinking SQL modes in two places. I think we should only link in the bottom paragraph

Reply:
removed first link